test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_CSharp.cs

```diff
@@ -43,6 +44,68 @@ public SamplesEndToEndTests_CSharp(TestFixture fixture)
             _settingsManager = ScriptSettingsManager.Instance;
         }
 
+        [Fact]
+        public async Task HostAdminApis_ValidAdminToken_Succeeds()
```

Comment:
Porting a couple tests that I previously wrote in v4 that don't exist in v3. Increases code coverage for these codepaths.